.github/actions/repo-gardening/src/tasks/check-description/index.js

```diff
@@ -0,0 +1,266 @@
+/**
+ * External dependencies
+ */
+const moment = require( 'moment' );
+
+/**
+ * Internal dependencies
+ */
+const debug = require( '../../debug' );
+const getLabels = require( '../../get-labels' );
+const getNextValidMilestone = require( '../../get-next-valid-milestone' );
+
+/* global GitHub, WebhookPayloadPullRequest */
+
+/**
+ * Check if a PR has unverified commits.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Promise<boolean>} Promise resolving to boolean.
+ */
+async function hasUnverifiedCommit( octokit, owner, repo, number ) {
+	let isUnverified = false;
+
+	for await ( const response of octokit.paginate.iterator( octokit.pulls.listCommits, {
+		owner: owner.login,
+		repo,
+		pull_number: +number,
+	} ) ) {
+		response.data.map( commit => {
+			if ( commit.commit.message.includes( '[not verified]' ) ) {
+				isUnverified = true;
+			}
+		} );
+	}
+
+	return isUnverified;
+}
+
+/**
+ * Check for status labels on a PR.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Promise<boolean>} Promise resolving to boolean.
+ */
+async function hasStatusLabels( octokit, owner, repo, number ) {
+	const labels = await getLabels( octokit, owner, repo, number );
+	let hasStatus = false;
+
+	// We're really only interested in status labels
+	labels.map( label => {
+		if ( label.includes( '[Status]' ) ) {
+			debug( `check-description: this label (${ label }) includes the word Status` );
+			hasStatus = true;
+		}
+	} );
+
+	return hasStatus;
+}
+
+/**
+ * Build a string with info about the next milestone.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Promise<string>} Promise resolving to info about the next release for that plugin.
+ */
+async function buildMilestoneInfo( octokit, owner, repo, number ) {
+	const labels = await getLabels( octokit, owner, repo, number );
+
+	// Find out what plugin we need to worry about.
```

Comment:
What would happen if the PR will touch multiple plugins? I don't think there would be PR's that would make production changes to multiple plugins at once, so it should not be big deal anyway.

Reply:
That's a good point, I didn't think about that. I have now updated the PR so we take that into account:

- if no plugin label is added, no milestone info will be added.
- if one or more plugin label is added, info about all the plugins is added, e.g.
![image](https://user-images.githubusercontent.com/426388/107791663-802c4f00-6d54-11eb-99a1-0163621146b7.png)